spec/rails_helper.rb

```diff
@@ -48,6 +48,13 @@ def visit_sandbox
   visit page_path(id: 'sandbox')
 end
 
+def click_download_panel
+  expect(page).to have_css('.sul-embed-download-panel', visible: false)
+  page.find('button[data-sul-embed-toggle="sul-embed-download-panel"]', visible: true)
+  page.find('[data-sul-embed-toggle="sul-embed-download-panel"]', match: :first).click
+  expect(page).to have_css('.sul-embed-download-panel', visible: true)
+end
```

Comment:
"kill it" as in ... move on and forget about it?  I'm ok w that.  Slight pref for rename, but it's ok w me as is.